scripts/check_msi_installation_files_regular_debug.txt

```diff
@@ -0,0 +1,262 @@
+C:\Program Files\ebpf-for-windows\testing\printk_legacy.sys
+C:\Program Files\ebpf-for-windows\testing\printk_unsafe.o
+C:\Program Files\ebpf-for-windows\testing\reflect_packet.o
+C:\Program Files\ebpf-for-windows\testing\reflect_packet.sys
+C:\Program Files\ebpf-for-windows\testing\run_tests.bat
+C:\Program Files\ebpf-for-windows\testing\sample_ebpf_ext.pdb
+C:\Program Files\ebpf-for-windows\testing\sample_ebpf_ext.sys
+C:\Program Files\ebpf-for-windows\testing\sample_ext_app.exe
+C:\Program Files\ebpf-for-windows\testing\sample_ext_app.pdb
+C:\Program Files\ebpf-for-windows\testing\sockops.o
+C:\Program Files\ebpf-for-windows\testing\sockops.sys
+C:\Program Files\ebpf-for-windows\testing\tail_call.o
+C:\Program Files\ebpf-for-windows\testing\tail_call.sys
+C:\Program Files\ebpf-for-windows\testing\tail_call_bad.o
+C:\Program Files\ebpf-for-windows\testing\tail_call_bad.sys
+C:\Program Files\ebpf-for-windows\testing\tail_call_bad_um.dll
+C:\Program Files\ebpf-for-windows\testing\tail_call_bad_um.pdb
+C:\Program Files\ebpf-for-windows\testing\tail_call_map.o
+C:\Program Files\ebpf-for-windows\testing\tail_call_map.sys
+C:\Program Files\ebpf-for-windows\testing\tail_call_max_exceed.o
+C:\Program Files\ebpf-for-windows\testing\tail_call_max_exceed.sys
+C:\Program Files\ebpf-for-windows\testing\tail_call_max_exceed_um.dll
+C:\Program Files\ebpf-for-windows\testing\tail_call_max_exceed_um.pdb
+C:\Program Files\ebpf-for-windows\testing\tail_call_multiple.o
+C:\Program Files\ebpf-for-windows\testing\tail_call_multiple.sys
+C:\Program Files\ebpf-for-windows\testing\tail_call_multiple_um.dll
+C:\Program Files\ebpf-for-windows\testing\tail_call_multiple_um.pdb
+C:\Program Files\ebpf-for-windows\testing\tail_call_recursive.o
+C:\Program Files\ebpf-for-windows\testing\tail_call_recursive.sys
+C:\Program Files\ebpf-for-windows\testing\tail_call_recursive_um.dll
+C:\Program Files\ebpf-for-windows\testing\tail_call_recursive_um.pdb
+C:\Program Files\ebpf-for-windows\testing\tail_call_sequential.o
+C:\Program Files\ebpf-for-windows\testing\tail_call_sequential.sys
+C:\Program Files\ebpf-for-windows\testing\tail_call_sequential_um.dll
+C:\Program Files\ebpf-for-windows\testing\tail_call_sequential_um.pdb
+C:\Program Files\ebpf-for-windows\testing\tail_call_um.dll
+C:\Program Files\ebpf-for-windows\testing\tail_call_um.pdb
+C:\Program Files\ebpf-for-windows\testing\test_sample_ebpf.o
+C:\Program Files\ebpf-for-windows\testing\test_sample_ebpf.sys
+C:\Program Files\ebpf-for-windows\testing\test_utility_helpers.o
+C:\Program Files\ebpf-for-windows\testing\test_utility_helpers_um.dll
+C:\Program Files\ebpf-for-windows\testing\test_utility_helpers_um.pdb
+C:\Program Files\ebpf-for-windows\testing\unit_tests.exe
+C:\Program Files\ebpf-for-windows\testing\unit_tests.pdb
+C:\Program Files\ebpf-for-windows\testing\usersim.dll
+C:\Program Files\ebpf-for-windows\testing\usersim.pdb
+C:\Program Files\ebpf-for-windows\testing\xdp_tests.exe
```

Comment:
since test EXEs are installed, shouldn't we also copy the setup scripts used by the CI to run them? eg. scripts\setup_ebpf_cicd_tests.ps1 etc. (in general the stuff copied by Export-BuildArtifactsToVMs)

Reply:
Since this PR only introduces the MSI test, the installer update can be done separately. Resolving this comment.
I created this issue to see if the team agrees with my reasoning:
https://github.com/microsoft/ebpf-for-windows/issues/2969